mapper/mapper.go

```diff
@@ -29,9 +29,10 @@ type TimeSlice []time.Time
 
 // Dimension names
 const (
-	DimensionTime      = "time"
-	DimensionAge       = "age"
-	DimensionGeography = "geography"
+	DimensionTime          = "time"
+	DimensionAge           = "age"
+	DimensionGeography     = "geography"
+	CurrentSixteensVersion = "67f6982"
```

Comment:
I think `SixteensVersion` would have been a better name. Have the name prefixed with `Current` implies that it's the most up to date which may not be the case